resource-aggregate/service/aggregate.go

```diff
@@ -220,6 +220,12 @@ func cleanUpToSnapshot(ctx context.Context, aggregate *aggregate, events []event
 			}
 			break
 		}
+		if ru, ok := event.(*raEvents.ResourceLinksSnapshotTaken); ok {
+			if err := aggregate.eventstore.RemoveUpToVersion(ctx, []eventstore.VersionQuery{{GroupID: ru.GroupId(), AggregateID: ru.AggregateId(), Version: ru.Version()}}); err != nil {
+				log.Info("unable to remove events up to snapshot for resource link of deviceId('%v')", ru.GetDeviceId())
```

Comment:
which resource link?

Reply:
renamed `"unable to remove events up to snapshot with version('%v') for resource links of deviceId('%v')", ru.Version(), ru.GetDeviceId())`